test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -367,6 +368,9 @@ private HapiApiSpec autoAccountCreationsHappyPath() {
 								"transferTxn")
 				).then(
 						/* get transaction record and validate the child record has alias bytes as expected */
+						getReceipt("transferTxn")
+								.andAnyChildReceipts()
+								.logged(),
```

Comment:
Do we need to assert the child receipt has `AccountID` ?

Reply:
yeah, would it be even better if we asserted the count.. i.e., the no of automatically created accounts ??